apps/web/src/pages/User/styles.ts

```diff
@@ -1,7 +1,17 @@
 import Image from 'next/image';
 import { styled } from 'stitches/stitches.config';
-export const StyledUser = styled('div', {});
-
+import { Button } from 'ui';
+export const StyledUser = styled('div', {
+});
+export const FollowButton = styled(Button, {
+  position: 'absolute',
```

Comment:
is position absolute necessary?
this might mess up in different screen sizes, could you try to make it work without it?

Reply:
same for the other components